pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -322,7 +324,10 @@ def phase_str(phi: float) -> str:
         if sampling_rate:
             curves = ys_mod if draw_output else ys
             yseff = ch_data.interpolate_curves(curves, sampling_rate)
-        ref_ys = yseff if sampling_rate else ys
+        ref_ys = [
+            list(chain.from_iterable(all_ys))
+            for all_ys in zip(ys, ys_mod, yseff)
+        ]
```

Comment:
They are being concatenated, not summed (the distinction is relevant since they are numpy arrays). So, where you write e.g. `ys.amp+ymod.amp+yseff.amp`, what you get is rather 
`[ys.amp[0], ...., ys.amp[N], ymod.amp[0], ... ymod.amp[N], yseff.amp[0], ..., yseff.amp[N]]`
This is what `chain` is doing for each quantity.

Reply:
Yes sorry I thought of them as lists and I also meant concatenation with my `+`. My question is more about what is done with this list `ref_ys`.

Now if I get it right, it is mainly used to get max and min of amplitude, detuning and phase, values that are then used to configure the limits of the y-axis. The automatic determination of the y-axis range in `Sequence.draw()` is then improved by taking the max and min over all the computed signals and not only the one being showed ?

Reply:
Well, not exactly. Note that `yseff` and `ys_mod` are only defined if they are going to be showed too. So what we are doing now is taking the `min` and `max` over all the _displayed_ signals, rather than just one of them.